### PR TITLE
CODER-95 redux persist overwrites

### PR DESCRIPTION
**Pull request by robinsiep**
## General information

**Ticket**

https://partypeak.atlassian.net/browse/CODER-95

## Description

Resolves the issue of the `redux-persist` subgenerator sometimes overwriting the work of the `redux-observable` generator. Also includes a fix for some console errors that occurred with a blank reducer.

## Additional notes
nope

_When creating a new PR be sure to fill in something under every heading, even if the answer is simply no._

### CODE REVIEW

File: __tests__/redux-persist.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+'use strict';
+const path = require('path');
+const assert = require('yeoman-assert');
+const helpers = require('yeoman-test');
+
+const storeCode = `\
+import { createStore } from 'redux';
+
```
Comment (rikvanderwerf) [inline __tests__/redux-persist.js:8]: white space

**Review by rikvanderwerf — CHANGES_REQUESTED** [review]

File: __tests__/redux-persist.js (hunk 1 above)
Comment (robinsiep) [reply __tests__/redux-persist.js:8]: Intentional, between third and first party imports. Kinda pythonesque :P

Comment (robinsiep) [reply __tests__/redux-persist.js:8]: @rikvanderwerf

**Comment by robinsiep** [comment]
It’d be nice to move this forward if possible @rikvanderwerf

**Pull request merged**